tests/testthat/test-gaussian.R

```diff
@@ -0,0 +1,20 @@
+context("gaussian regression")
+
+test_that("gaussian regression with dense features work", {
+  x <- matrix(rnorm(30), 10, 3)
```

Comment:
would be great if you could use real data rather than (or in addition to) simulated data for the tests